providers/os/resources/packages/windows_packages.go

```diff
@@ -203,15 +204,78 @@ func (w *WinPkgManager) Format() string {
 	return "win"
 }
 
-const installedAppsScript = `
-Get-ItemProperty (@(
-  'HKLM:\\SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Uninstall\\*',
-  'HKCU:\\SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Uninstall\\*',
-  'HKLM:\\SOFTWARE\\Wow6432Node\\Microsoft\\Windows\\CurrentVersion\\Uninstall\\*',
-  'HKCU:\\SOFTWARE\\Wow6432Node\\Microsoft\\Windows\\CurrentVersion\\Uninstall\\*'
-) | Where-Object { Test-Path $_ }) |
-Select-Object -Property DisplayName,DisplayVersion,Publisher,EstimatedSize,InstallSource,UninstallString | ConvertTo-Json -Compress
-`
+func (w *WinPkgManager) getInstalledApps() ([]Package, error) {
+	pkgs := []string{
```

Comment:
Maybe I am missing something here, but how would that work with remote ssh scan of a Windows machine?

Reply:
i had missed that. we need both in that case, determining which one to use based on the connection type. i will fix this

Reply:
@chris-rock added that back. we now run directly through the registry when using `local` conn and as a fallback: we still use the powershell

Reply:
Perfect, thank you